CRM/Utils/Check/Component/Security.php

```diff
@@ -364,7 +364,7 @@ public function isDirAccessible($dir, $url) {
    * @return string
    */
   public function createDocUrl($topic) {
-    return CRM_Utils_System::getWikiBaseURL() . $topic;
+    return CRM_Utils_System::docURL2('sysadmin/setup/security#' . $topic, TRUE);
```

Comment:
```suggestion
    return CRM_Utils_System::docURL2('sysadmin/setup/security#' . $topic, TRUE);
```
where is "docurl2" defined as a function?

Reply:
@MikeyMJCO https://github.com/civicrm/civicrm-core/blob/master/CRM/Utils/System.php#L1288